app.rb

```diff
@@ -0,0 +1,172 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+
+class App
+  attr_accessor :book_list, :people
+
+  def initialize
+    @book_list = []
+    @people = []
+    @students = []
+    @teachers = []
+    @rentals = []
+  end
+
+  def run
+    user_input
+  end
+
+  def create_book
+    puts 'Enter BOOK Title:'
+    title = gets.chomp
+    puts 'Enter BOOK Author:'
+    author = gets.chomp
+    book = Book.new(title, author)
+    puts "The book '#{title.upcase}' by #{author.upcase} was created successfully!"
+    @book_list << book unless @book_list.include?(book)
+  end
+
+  def create_rental
+    if @book_list.empty?
+      puts 'Please ADD some books...'
+    elsif @people.empty?
+      puts 'Please ADD some people...'
+    else
+      puts "Select a book from the following list by number:\n"
+      list_all_books
+      index_of_book = gets.chomp.to_i
+      puts "Select a person from the following list by number:\n"
+      list_all_people
+      index_of_person = gets.chomp.to_i
+      rental = Rental.new(people[index_of_person], book_list[index_of_book])
+      puts 'Rental was Successfully Created!'
+      @rentals << rental unless @rentals.include?(rental)
+    end
+  end
```

Comment:
Great job so far @shahira-sadat 
- [ ] Please your app throws an error when the User wants to `Create a Renter`
![five](https://user-images.githubusercontent.com/55596970/193333557-27122111-67ff-4b87-88cc-f14591af9b31.png)